app/routes/application.js

```diff
@@ -1,6 +1,7 @@
 import Route from '@ember/routing/route';
 import { service } from '@ember/service';
 import ENV from 'frontend-gelinkt-notuleren/config/environment';
+import { decentLocaleMatch } from '../utils/intl';
```

Comment:
My intent was that we would import the function from the editor repo rather than having to duplicate it in every codebase. Is there a good reason not to do that?

Reply:
Yeah I see, the main reason I did not import it from the editor is that the function is not really inherent to the editor codebase (it is quite generic).